doc/example.js

```diff
@@ -0,0 +1,32 @@
+var React = require('react');
```

Comment:
Done, but is there a way to have make-up lint just the file? I've had to add an `.eslintignore` to stop it finding 15k linting errors in the browserified build